crates/ruff_python_ast/src/nodes.rs

```diff
@@ -971,12 +971,19 @@ impl Ranged for FStringExpressionElement {
     }
 }
 
+/// An `FStringLiteralElement` with an empty `value` is an invalid f-string element.
```

Comment:
This is fine because there can never be an empty `FStringLiteralElement` for any valid f-string including the ones which only contains expressions. This is because the lexer doesn't emit any `FStringMiddle` token according to the PEP.